src/UI/Component/Item/Item.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Item;
+
+/**
+ * Common interface to all items.
+ */
+interface Item extends \ILIAS\UI\Component\Component {
+	/**
+	 * Gets the title of the item 
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Create a new item with an attached description.
+	 * @param string $description
+	 * @return AppointmentItem
+	 */
+	public function withDescription($description);
+
+	/**
+	 * Get the description of the item.
+	 * @return string
+	 */
+	public function getDescription();
+
+	/**
+	 * Get a new item with the given properties as key-value pairs.
+	 *
+	 * The key is holding the title and the value is holding the content of the
+     * specific data set.
+	 *
+	 * @param array<string,string> $properties Title => Content
+	 * @return self
+	 */
+	public function withProperties(array $properties);
+
+	/**
+	 * Get the properties of the appointment.
+	 *
+	 * @return array<string,string>		Title => Content
+	 */
+	public function getProperties();
+
+	/**
+	 * Create a new appointment item with a set of actions to perform on it.
+	 *
+	 * @param string[] $actions
+	 * @return AppointmentItem
+	 */
+	public function withActions(array $actions);
+
+	/**
+	 * Get the actions of the item.
+	 *
+	 * @return string[]
+	 */
+	public function getActions();
+
+	/**
+	 * Set a marker id. Marker IDs must be >=0 and <=32. They assign the item to a CSS class il-item-marker-[ID].
+	 * Set 0 for no marker (default).
+	 *
+	 * @param int $marker_id
+	 */
+	public function withMarkerId($marker_id);
+
+	/**
+	 * @return int
+	 */
+	public function getMarkerId();
+
+	/**
+	 * Set image as lead
+	 *
+	 * @param \ILIAS\UI\Component\Image\Image $image lead image
+	 */
+	public function withLeadImage(\ILIAS\UI\Component\Image\Image $image);
```

Comment:
Yes. Currently withLeadImage or withLeadText. Certainly also extendable by adding a withLeadIcon.

Currently all as options in the Standard List Item, since I am not sure if our UI really should have knowledge about Blogs, Forums or similar things, if a generic approach is possible. Btw blog postings certainly could make use of a Lead Image.